### ISSUE TITLE
Yokai Watch 3 Localize in Feb 8 2019 

### ISSUE THREAD

**Issue by Orionsilver** [togenyan/Yo-kai_Editor_3#43]
Finally it is #LocalizeYW3 
https://twitter.com/NintendoAmerica/status/1045300497970843648
![yw3 locolize](https://user-images.githubusercontent.com/18640392/46153940-26ab4e80-c285-11e8-9fa4-02a2a0df791f.png)

it is time to prepare for the English version YW3 Save editor

**Comment by Conan179** [comment]
When i load the sav from the german eshop version, its show this.
I cant read jap.
![yokai3_eu_ge](https://user-images.githubusercontent.com/1906242/49689533-be010d80-fb22-11e8-8001-b34a6783eec6.JPG)
i thinnk, the read was succes, only eng lang. missing.

**Comment by justin102955** [comment]
> When i load the sav from the german eshop version, its show this.
> I cant read jap.
> ![yokai3_eu_ge](https://user-images.githubusercontent.com/1906242/49689533-be010d80-fb22-11e8-8001-b34a6783eec6.JPG)
> i thinnk, the read was succes, only eng lang. missing.

Yeah. I think all that's needed now is the English patch to this. Nonetheless, it still works!

**Comment by Orionsilver** [comment]
Yokai Watch 3 different language 
Europe - Fr 
Europe - En
Europe - De
Europe - It
Europe - Es
USA - En
Canada - Fr
Latin America - Es
Korean - Ko.

North America on February 8, 2019
Europe on December 7 2018 
@togenyan

**Comment by Orionsilver** [comment]
The game is based on Yokai Watch 3 Sukiyaki

**Comment by Orionsilver** [comment]
Check here:
http://yokaiwatch.wikia.com/wiki/List_of_Yo-kai_by_Medallium_Number_(Yo-kai_Watch_3)
http://yokaiwatch.wikia.com/wiki/Items
@togenyan @eeelkababs @Conan179 @justin102955 @bui
